### DISCUSSION TITLE
when I click osu! gameplay resume cursor this's not in high precision mode(During gameplay?)

### DISCUSSION THREAD

**Discussion by cdwcgt** [ppy/osu#14281] [Q&A] [answered]
When I set `Confine mouse cursor to window` to `During Gameplay`
it seen only enable `high precision` mouse mode during gameplay
but when I pause and need to click resume cursor(Orange cursor), `high precision mouse is not enable so when I click and resume the cursor will teleport(if I use tablet) because high precision mouse enable.

**Comment by peppy** [comment, marked as answer]
Your question makes no sense sorry. Could you please attach a video or try to explain what you mean again?

**Reply by cdwcgt** [reply]
https://user-images.githubusercontent.com/34775378/131083640-281f1300-f441-4968-8dc7-e3027cb7b00a.mp4

when I pause it change my tablet mapping like turn off `high precision`

**Reply by peppy** [reply]
This looks like an issue with your tablet drivers or software. It is providing cursor positions that are overriding raw input.

What tablet drivers / software are you using? Also is there any reason you can't use the in-game configuration? That should avoid issues like this.

**Reply by cdwcgt** [reply]
I use [TabletDriver](https://github.com/hawku/TabletDriver) and here is my settings
![image](https://user-images.githubusercontent.com/34775378/131150108-afe5a028-a166-4533-843b-53f970c6d4bb.png)
in-game configuration means this?
![image](https://user-images.githubusercontent.com/34775378/131150293-e353425f-be78-4ca1-b9d2-3ca9da4542d4.png)

**Reply by frenzibyte** [reply]
> in-game configuration means this?
> ![image](https://user-images.githubusercontent.com/34775378/131150293-e353425f-be78-4ca1-b9d2-3ca9da4542d4.png)

Yes, and as pointed in that note in yellow, read the FAQ for troubleshooting the issue about your tablet not being detected.

**Reply by cdwcgt** [reply]
![image](https://user-images.githubusercontent.com/34775378/131207812-9e8aa090-e303-4e57-9a54-89ac6b6f08eb.png)
ok I think this problem is solved

(but it seen like I need to reinstall driver every times when I need to use tablet to draw or osu, Is there a solution

**Reply by frenzibyte** [reply]
Reinstall? Isn't what you need is just turning off that external driver when running osu!lazer? also have you tried installing the [system-wide OpenTabletDriver app](https://github.com/OpenTabletDriver/OpenTabletDriver/releases/tag/v0.5.3.3) rather than the one you're currently using? It may work better and allow osu!lazer to take over input without requiring you to exit the system-wide app.

**Reply by cdwcgt** [reply]
because I use tabletdriver for other program(because he can read the pressure and windows ink) and its need install its driver and conflict with OpenTabletDriver so I need to reinstall driver
